cmd/src/batch_remote.go

```diff
@@ -78,8 +78,15 @@ Examples:
 		}
 		ui.ResolvingNamespaceSuccess(namespace.ID)
 
+		ui.SendingBatchChange()
+		batchChangeName, err := svc.UpsertBatchChange(ctx, spec.Name, namespace.ID)
+		if err != nil {
+			return err
+		}
+		ui.SendingBatchChangeSuccess()
+
 		ui.SendingBatchSpec()
-		batchSpecID, err := svc.UpsertBatchSpecInput(
+		batchSpecID, err := svc.CreateBatchSpecFromRaw(
```

Comment:
@LawnGnome Had to switch this to `CreateBatchSpecFromRaw` because `UpsertBatchSpecInput` tries to delete the last spec belonging to the batch change, and that results in an error because of the FOREIGN constraint on `batch_changes_batch_spec_id_fkey`.

This is closely similar to what we have in the UI currently.

Reply:
That's fine for me.